### PR TITLE
GCW-2617  breadcrumbs for browse

### PR DESCRIPTION
**Pull request by abhinav098**
Hello Team this PR is for adding breadcrumbs on browse. 
Please review thanks.

### CODE REVIEW

**Comment by abulsayyad123** [comment]
Resolve the conflicts.

File: app/controllers/application.js [hunk 1]
```diff
@@ -30,8 +30,8 @@ export default Ember.Controller.extend({
     return /Android/i.test(navigator.userAgent) && !this.get("isMobileApp");
   }),
 
-  isAndroidBrowser: Ember.computed("isMobileApp", function() {
-    return /Android/i.test(navigator.userAgent) && !this.get("isMobileApp");
+  isBrowsePage: Ember.computed("currentPath", function() {
+    return ["browse", "package_category"].indexOf(this.get("currentPath")) >= 0;
```
Comment (patrixr) [inline app/controllers/application.js]: how come `isBrowsePage` is true if the path is `package_category` ? 🤔

File: app/routes/home.js [hunk 2]
```diff
@@ -6,13 +6,15 @@ export default PublicRoute.extend({
...
+    applicationController.set("pageTitle", this.get("i18n").t("browse.title"));
     applicationController.set("isHomePage", true);
   },
 
   resetController: function(controller, isExiting) {
     this._super.apply(this, arguments);
     if (isExiting) {
-      this.controllerFor("application").set("isHomePage", false);
+      let controller = this.controllerFor("application");
+      controller.set("isHomePage", false);
```
Comment (patrixr) [inline app/routes/home.js]: this is no longer needed, the `isHomePage` property is now a computed property in the application controller : 

```javascript
isHomePage: Ember.computed("currentPath", function() {
    return this.get("currentPath") === "home";
  })
```

Comment (abhinav098) [reply app/routes/home.js]: Yeah, I'm removing it.

File: app/controllers/application.js (hunk 1 above)
Comment (abhinav098) [reply app/controllers/application.js]: browse_related i meant.. by this, will change the name.

**Comment by abhinav098** [comment]
merging this.

**Pull request merged**